### PR TITLE
Added sorting

### LINKED ISSUE #3: [BUG] Sorting is not working
Click on the heading of the column. It should sort the board.

### PR DESCRIPTION
**Pull request by im-adithya**
# Fixes 'Sorting not working' issue
## Bug Description
Sorting is not working on clicking the head of any column.

## Root cause
The sort function was done in the order of mergedPRs, openPRs and issues regardless of the sort parameter.

### Solution Description
I used three variables to specify our preference of criteria using switch and set the previous preference as default. Then I used those variables to compare for the sort function.

closes #3 

https://user-images.githubusercontent.com/64399555/105606591-5ff61980-5dc0-11eb-99ca-a3f81b692f50.mp4

### CODE REVIEW

**Comment by im-adithya** [comment]
@lolimay Can you please review this? :D

File: src/index.js [hunk 1]
```diff
@@ -23,29 +23,50 @@ function refreshTable(newData){
     const totalEm = document.querySelector('.total')
     totalEm.innerText = 'Total: ' + totalNumbers
 
-    contributors = contributors.sort( (a, b) => {
-        if ( a.mergedPRsNumber < b.mergedPRsNumber ){
+    contributors = contributors.sort((a, b) => {
+        var first, second, third // criteria is specified here
```
Comment (djcruz93) [inline src/index.js:27]: @im-adithya what you think of naming these variables as preference1, preference2, and preference3.? Or something which may be easier for someone new to understand the logic behind it.

**Review by djcruz93 — CHANGES_REQUESTED** [review]
@im-adithya logic looks good to me. It would be great if you could please share a screen record with the updated changes you have made, showing the working functionality. :smiley:

File: src/index.js (hunk 1 above)
Comment (im-adithya) [reply src/index.js:27]: Sure! Will do it.

Comment (im-adithya) [reply src/index.js]: @djcruz93 I've uploaded the video now. Please review it :)

**Review by djcruz93 — APPROVED** [review]

**Pull request merged**